mlflow/server/js/.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  printWidth: 100,
+  trailingComma: 'es5',
+  tabWidth: 2,
+  semi: true,
+  singleQuote: true,
+  arrowParens: 'always',
```

Comment:
|Rule|Description|
|:-|:-|
|`printWidth`|Specify the line length that the printer will wrap on.|
|`trailingComma`|Print trailing commas wherever possible when multi-line|
|`tabWidth`|Specify the number of spaces per indentation-level.|
|`singleQuote`|Use single quotes instead of double quotes.|
|`arrowParens`|Include parentheses around a sole arrow function parameter.|

https://prettier.io/docs/en/options.html#tab-width